addons/account/tests/test_sequence_mixin.py

```diff
@@ -367,6 +367,18 @@ def test_resequence_clash(self):
             'first_name': '2',
         }).resequence()
 
+    def test_resequence_with_blank_name(self):
+        """When a blank name is used, Resequence does not clash."""
+        moves = self.env['account.move']
+        for i in range(3):
+            moves += self.create_move(name=str(i))
+        moves[:1].name = False
```

Comment:
```suggestion
        for name in ['MOVE/01/0001', False, '/']:
            moves += self.create_move(name=name)
```
Like that you test all the use cases

Reply:
@FlorianGilbert Oh, thank you :D

Reply:
So, will you make the change ? :)

Reply:
> So, will you make the change ? :)

Yes, i'm busy at work i will try to fix it today to make it perfect

Reply:
Your proposal is excellent.

Reply:
@FlorianGilbert i tried as you suggested and
raise ValidationError(_( odoo.exceptions.ValidationError: The Date (01/01/2016) doesn't match the Number (MOVE/01/0001). You might want to clear the field Number before proceeding with the change of the date.

Reply:
@hoangtiendung070797 Have a look on `test_journal_sequence_multiple_type`, it could help you :wink:

Reply:
@FlorianGilbert Okay, let me make a reference to it. Tkank you

Reply:
@FlorianGilbert i fixed it. Please review it. Thank you :D